modules/Classes.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+// Creat Book class
+class Book {
+  constructor(title, author) {
+    this.bookid = Math.random().toFixed(1);
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Library {
+  constructor() {
+    this.books = [];
+  }
+
+  // Add a new book
+  addBook(newItem) {
+    this.books.push(newItem);
+    localStorage.setItem('DB', JSON.stringify(this.books));
+    Display(newItem);
+  }
+
+  // remove a book
+  remove(bookid) {
+    const rm = document.getElementById(bookid);
+    rm.remove();
+    this.books = this.books.filter((x) => x.bookid !== bookid);
+    localStorage.setItem('DB', JSON.stringify(this.books));
+  }
+}
+const storeBook = new Library();
+
+// fetch values from input
+const getInput = () => {
+  const title = document.querySelector('#book-title').value;
+  const author = document.querySelector('#book-author').value;
+  const addItem = new Book(title, author);
+  return addItem;
+};
+// Showcase the list of books
+let Display = (index) => {
```

Comment:
- [ ] You did a great job using ES6 arrow functions! 💯 However, please put special attention to the names of your variables/functions: By convention, capitalized names are reserved for classes. please consider fixing this one! 🤓 🛠️